string joint_name
int32 motor_position
int32 motor_speed